### ISSUE TITLE
Suggestion: block icons

### ISSUE THREAD

**Issue by joepvtl** [PerplexDigital/Perplex.ContentBlocks#46]
We only use 1 category in the content blocks, so all blocks get the same icon. I think it would be a good addition to be able to configure block icons to improve the user experience.

**Comment by PerplexDaniel** [comment]
Yeah agree, we could add some `Icon` property to the `IContentBlockDefinition` and use that. Category icon could be used as a fallback.

**Comment by PerplexDaniel** [comment]
Closing due to inactivity
